test/e2e/common/node/runtimeclass.go

```diff
@@ -51,6 +51,7 @@ var _ = SIGDescribe("RuntimeClass", func() {
 		handler := f.Namespace.Name + "-handler"
 		rcName := createRuntimeClass(f, "unconfigured-handler", handler)
```

Comment:
isn't it this call that actually makes the runtime class? why are we only deferring after creating the pod?

Reply:
I have adjusted the position, please check again. Thanks.

Reply:
I have adjusted the position, please check again. Thanks.